### PR TITLE
Implement Prepare step.

### PR DESCRIPTION
**Pull request by pvalena**
Depends on #31.

### CODE REVIEW

File: tmt/steps/prepare/__init__.py [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+        """ Show discover details """
+        self.super.show(keys = ['how', 'script'])
+
+    def go(self):
+        """ Prepare the test step """
+        self.super.go()
+        self.plan.provision.prepare(self.data['how'], self.data['script'])
+
+        # TODO: EXECUTE:
+        #   => self.plan.provision.execute("nohup", "run.sh", "..", ">generalout", "2>generalerr, "&"):
```
Comment (pvalena) [inline tmt/steps/prepare/__init__.py]: Oops, this should probably go to  `steps/__init__.py` (_well, it's just a suggestion_).

Comment (thrix) [reply tmt/steps/prepare/__init__.py]: @pvalena  why would we run.sh in prepare??

Comment (pvalena) [reply tmt/steps/prepare/__init__.py]: That was just a note for me to do that in execute. Please disregard that, as I've already PRs on the way wich fixes this, and implemets it on proper place.

**Comment by pvalena** [comment]
Also set default to Fedora again.

**Comment by pvalena** [comment]
Closin in favor of https://github.com/psss/tmt/pull/30

**Pull request closed without merging**